core/arch/arm/plat-imx/main.c

```diff
@@ -115,6 +115,21 @@ void console_init(void)
 #endif
 }
 
+static const char *const reserved[] = {
+#if defined(CFG_NXP_CAAM)
+#if defined(CFG_MX8MM) || defined(CFG_MX8MQ) || defined(CFG_MX8MP)
+	"sec_jr0",
```

Comment:
The job ring to disable is not always `jr0`. The job ring to disable is `CFG_JR_INDEX` defined in `crypto_conf.mk`